### ISSUE TITLE
NUNIT1003 false positive

### ISSUE THREAD

**Issue by chinwobble** [nunit/nunit.analyzers#312]
When you create test case that accepts an array it then it fails with NUNIT1003 incorrectly.

```csharp

[TestCase(null)]
[TestCase(new byte[] { })]
public async Task Test_Array(byte[] bytes)
{
    // assert ...
}
```
This analyzer should only fail if `new object[] { }` is supplied.

**Comment by manfred-brands** [comment]
I cannot repeat this here, I get no NUnit1003 for neither of the two attributes,  What version of nunit.analyzer are you using?

**Comment by chinwobble** [comment]
Here are the versions I'm using 
```xml
<PackageReference Include="NUnit" Version="3.11.0" />
 <PackageReference Include="Nunit.analyzers" Version="0.5.0" />
 <PackageReference Include="NUnit3TestAdapter" Version="3.12.0" />
```

**Comment by manfred-brands** [comment]
Yes, version 0.5.0 is broken, it works in 0.4.0 and it has been fixed in the current master branch.

**Comment by mikkelbu** [comment]
Sorry about the inconvenious @chinwobble. I think @manfred-brands is right about this duplicating #302. I'll see if I have time to make a release this week, as we already have solved some bugs in the current master.

**Comment by manfred-brands** [comment]
@mikkelbu Have you ever thought of having releases created automatically as soon as something is merged into master? That way users don't have to wait for a manual operation and it would save you time as well.

**Comment by chinwobble** [comment]
thanks hopefully it can be released soon.

**Comment by mikkelbu** [comment]
@manfred-brands We have prereleases for each build on master, but they are on myget - https://www.myget.org/feed/nunit-analyzers/package/nuget/NUnit.Analyzers - for historical reasons (as I mimicked the build scripts etc. after how they were structured in the NUnit repo - and possibly also the Console repo.

Perhaps we should consider to have both the regular release and the prereleases on nuget.org, but I've not had time to look into this.
